src/main/app/src/app/informatie-objecten/informatie-object-nieuwe-versie-signaler.ts

```diff
@@ -0,0 +1,7 @@
+import {Injectable} from '@angular/core';
+import {BehaviorSubject} from 'rxjs';
+
+@Injectable()
+export class InformatieObjectNieuweVersieSignaler {
+    public isNieuweVersieBeschikbaar: BehaviorSubject<boolean> = new BehaviorSubject<boolean>(false);
+}
```

Comment:
Dit doe met eigenlijk denken aan websockets. Hebben we nog geen websockets op dit scherm? Want als je dit soort dingen met websockets doet dan werkt het ook als iemand anders een nieuwe versie toevoegt. En we hebben een ScreenEvent voor als een informatieobject geupdate wordt. Die wordt vanuit de open_notificaties die we krijgen aangemaakt en verstuurd.

Reply:
Ja we hebben op die pagina een websocket op het enkelvoudiginformatieobject, die getriggered wordt op een update van het document. Maar die update niet de documenviewer. 

Maar goed punt dat hij ook moet updaten als iemand anders een document toevoegt, dan wil je inderdaad dat hij voor iedereen geupdate wordt, dus ik zal nog even in de websockets duiken, wellicht dat ik die moet toevoegen aan de documentviewer.

Reply:
Omgezet naar websockets